### ISSUE TITLE
Text normalization

### ISSUE THREAD

**Issue by TianduoWang** [anhnh2002/XTTSv2-Finetuning-for-New-Languages#26]
Hi,

Thanks for your great work!

May I check with you how you normalize the text for Vietnamese? I notice that in the TTS code that provided in your repo, only basic normalization (like lower case and whitespace cleaning) are applied. I wonder whether this is enough for fine-tuning XTTS on Vietnamese.

**Comment by anhnh2002** [comment]
Yes, I have cleaned it into metadata.csv, so I don’t need to clean it anymore.
